buildSrc/src/main/java/com/epmedu/animeal/configure/AppPluginSection.kt

```diff
@@ -37,4 +37,19 @@ internal fun Project.configureAndroidApplication() = applicationExtension.run {
             isMinifyEnabled = false
         }
     }
+
+    flavorDimensions += "environment"
+    productFlavors {
```

Comment:
With this changes we end up with 6 new builds (devDebug, devRelease, devRelease, prodRelease and so on) what will be the usage/purpose of each one?

Reply:
To be flexible in case we need to debug different environments and check release version as well.
QA team will be invited to test release versions with testing and prod environments. We have a separate development environment, but we need 2 types: debug to connect debugger and release to verify that obfuscating doesn't break queries. Looks like that we will use only devDebug and devRelease, while QA team will use qaRelease and prodRelease. Why should we avoid having a possibility to build debug versions for last 2 though?

Reply:
Makes sense, I'm thinking about document this description somewhere (Animeal KB or EPMEDU-2491) just to specify what would be the new flavors and the usage of each.